SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/DuoSamplesListFragment.kt

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.Licensed under the MIT License.
+ */
+
+package com.microsoft.device.display.samples.duosamples
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.fragment.app.Fragment
+import com.microsoft.device.display.samples.companionpane.CompanionPaneActivity
+import com.microsoft.device.display.samples.draganddrop.DragAndDropActivity
+import com.microsoft.device.display.samples.dualview.DualViewActivity
+import com.microsoft.device.display.samples.extendcanvas.ExtendedCanvasActivity
+import com.microsoft.device.display.samples.hingeangle.HingeAngleActivity
+import com.microsoft.device.display.samples.intentsecondscreen.IntentToSecondScreenFirstActivity
+import com.microsoft.device.display.samples.listdetail.ListDetailsActivity
+import com.microsoft.device.display.samples.multipleinstances.MultipleInstancesMainActivity
+import com.microsoft.device.display.samples.pen.PenEventsActivity
+import com.microsoft.device.display.samples.qualifiers.QualifiersActivity
+import com.microsoft.device.display.samples.twopage.TwoPageActivity
+
+/**
+ * A simple [Fragment] subclass as the default destination in the navigation.
+ */
+class DuoSamplesListFragment : Fragment() {
+
+    companion object {
+        private const val GIT_LINK_DRAG_AND_DROP = "https://github.com/microsoft/surface-duo-sdk-samples-kotlin/tree/main/DragAndDrop"
```

Comment:
I think that it's a good idea to move all links to a new class, something like GithubRepoLinks